apps/redi-connect/src/pages/app/profile/Profile.tsx

```diff
@@ -205,14 +203,16 @@ function Profile() {
                   </Columns.Column>
                 )}
                 <Columns.Column>
-                  <ReadRediClass.Some profile={profile} />
+                  {/* Commented until we implement it using the data available in Salesforce */}
+                  {/* <ReadRediClass.Some profile={profile} /> */}
+                  <ReadOccupation.Some profile={profile} />
                 </Columns.Column>
               </Columns>
             </Element>
           )}
 
-          {(profile.mentor_occupation ||
-            profile.mentee_occupationCategoryId) && (
+          {profile.mentor_occupation && (
+          // || profile.mentee_occupationCategoryId) 
```

Comment:
Is this comment also intended or left here by mistake? Asking because I'm not sure if occupationCategoryId is related to ReDI courses

Reply:
@helloanil, I left it intentionally because I put `ReadOccupation` in place of hidden `ReadRediClass` and it affects the mentee UI. We would need to uncomment it after `ReadRediClass` is re-implemented.